### ISSUE TITLE
Training ProtoNet from scratch can only attain ~29.5% acc

### ISSUE THREAD

**Issue by leodmel** [Sha-Lab/FEAT#2]
Hello, I can only attain 29.44% accuracy when running `train_protonet.py` by the default setting, however, ProtoNet should get about 49.42% accuary according to the paper, do you run `train_protonet.py` by its default setting or should I set some hyperparameters manually? Any idea would be appreciated.

**Comment by Han-Jia** [comment]
Hello, some default parameters are used for the pre-trained model. Please try the learning rate (lr) 0.001 or 0.002, and set step_size/gamma as 20/0.5. 

If you run with the parameters above, you can get 1-shot 5-way performance at around 45%. The 49% performance shown in the ProtoNet paper uses the 30-way meta-training for the 5-way test.

**Comment by leodmel** [comment]
@Han-Jia Thanks a lot!
